templates/deployment.yaml

```
@@ -52,9 +52,12 @@ spec:
           volumeMounts:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.extraVolumeDevices }}
+          volumeDevices:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-
       {{- with .Values.extraVolumes }}
```

```suggestion
      {{- with .Values.extraVolumes }}
```

`extraVolumeDevices` needs to be added to this section as well; would there ever be a need to have both `extraVolumes` and `extraVolumeDevices`? or can we just assume that only one list will exist?

---

```suggestion
        volumes:
        {{- with .Values.extraVolumes }}
          {{- toYaml . | indent 8 }}
        {{- end }}
        {{- with .Values.extraVolumeDevices }}
          {{- toYaml . | indent 8 }}
        {{- end }}
```

this might work?